restructuredtext_lint/test/test.py

```diff
@@ -93,7 +96,30 @@ def test_invalid_link(self):
 
         This is a regression test for https://github.com/twolfson/restructuredtext-lint/issues/12
         """
-        filepath = __dir__ + '/test_files/invalid_link.rst'
+        filepath = rst_dir + '/invalid_link.rst'
         errors = restructuredtext_lint.lint_file(filepath)
         self.assertIn('Anonymous hyperlink mismatch: 1 references but 0 targets.', errors[0].message)
         self.assertIn('Hyperlink target "hello" is not referenced.', errors[1].message)
+
+
+class TestRestructuredtextLintCLI(TestCase):
+    """ Tests for 'rst-lint' CLI comand """
+
+    def test_rst_lint_filepaths_not_given(self):
```

Comment:
For consistency with the rest of these tests, please add a docstring to each of the functions.